src/utils/narrow.js

```diff
@@ -537,3 +537,24 @@ export const getNarrowForReply = (message: Message | Outbox, ownUserId: UserId)
     return topicNarrow(streamName, message.subject);
   }
 };
+
+/**
+ * Answers, this question, "What should be the right accessibilityLabel for
+ * send message button?"
+ *
+ * This function only provide labels for stream, topic and pm narrow.
+ * For other narrows, there is no point of label.
+ */
+export function narrowUiDescription(narrow: Narrow, _: GetText): string {
+  return caseNarrow(narrow, {
+    stream: name => _('Narrow: stream {streamName}', { streamName: name }),
+    topic: (streamName, topic) =>
+      _('Narrow: stream {streamName}, topic {topic}', { streamName, topic }),
+    pm: () => _('Narrow: PM'),
```

Comment:
This should list everyone in the PM conversation, as I mentioned [here](https://chat.zulip.org/#narrow/stream/48-mobile/topic/Adding.20Accessibilty/near/1130272).